source/manual/fall-back-to-mirror.html.md

```diff
@@ -26,15 +26,15 @@ We use multiple mirrors across various AWS regions and GCP to ensure redundancy
 
 If [Fastly, our primary CDN](/manual/cdn.html), cannot fetch a page from our backend servers (becuase of a timeout or a 5xx error), then Fastly will attempt to serve a page from a mirror in order of priority.
 
-## How are the mirrors populated
+## How are the mirrors populated?
 
-Every day the [govuk-mirror-sync] cronjob crawls the www and assets domains, saves pages and assets to disk and then uploads the files to the primary S3 bucket. The [govuk-mirror] repository contains the code responsible for crawling and saving pages to disk.
+Every day the [govuk-mirror-sync cronjob (configured in govuk-helm-charts)][govuk-mirror-sync configuration] crawls the www and assets domains, saves pages and assets to disk and then uploads the files to the primary S3 bucket. The [govuk-mirror] repository contains the code responsible for crawling and saving pages to disk.
```

Comment:
Suggest you don't need the parenthetical here - we didn't used to say "configured in govuk-puppet" for everything configured in puppet, and lots of things are configured there so it would be noisy to say this every time. The link already makes it clear where it's configured.

```suggestion
Every day the [govuk-mirror-sync cronjob][govuk-mirror-sync configuration] crawls the www and assets domains, saves pages and assets to disk and then uploads the files to the primary S3 bucket. The [govuk-mirror] repository contains the code responsible for crawling and saving pages to disk.
```

Reply:
Thanks - I agree. My aim was to differentiate between the job configuration, and the job in Argo (they could both be called "govuk-mirror-sync cronjob", with the destination unclear until you click on the link.

I'll give it another tweak before merging.